src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.49.0"
+__version__ = "0.50.0"
```

Comment:
@johnyrahul @gaya3-zipstack this can be `0.49.0` itself. We have not released that version of the SDK yet
```suggestion
__version__ = "0.49.0"
```